Sources/AppleAPI/URLRequest+Apple.swift

```diff
@@ -117,4 +119,14 @@ extension URLRequest {
     static var olympusSession: URLRequest {
         return URLRequest(url: .olympusSession)
     }
+
+    /// Federate the sign in to get the X-Apple-HC header keys in order to properly mint a hashcash during regular signin
+    static func federate(account: String, serviceKey: String) throws -> URLRequest {
+        var request = URLRequest(url: .signIn)
+        request.allHTTPHeaderFields?["Accept"] = "application/json"
+        request.allHTTPHeaderFields?["Content-Type"] = "application/json"
+        request.httpMethod = "GET"
+        
+        return request
+    }
```

Comment:
@MattKiazyk can you explain me what would the Request look like in this case ?
Would the account and serviceKey be included as query parameters ?